test/validation/api/ipsec/ipsec.c

```diff
@@ -815,26 +947,46 @@ static int ipsec_send_out_one(const ipsec_test_part *part,
 			ev = recv_event(queue, 0);
 			if (ODP_EVENT_INVALID != ev) {
 				CU_ASSERT(odp_event_is_valid(ev) == 1);
+
 				CU_ASSERT_EQUAL(ODP_EVENT_PACKET,
 						odp_event_types(ev, &subtype));
-				CU_ASSERT_EQUAL(ODP_EVENT_PACKET_BASIC,
-						subtype);
-				CU_ASSERT(!part->out[i].status.error.all);
+
+				if (ODP_EVENT_PACKET_IPSEC != subtype) {
```

Comment:
This condition should always be true. At this point we are reading the pktio queue of the of the loopback interface to which we are sending inline-processed packets. So the only thing we are supposed to receive here are plain packets.

Reply:
Ack. Reverting this.